x/emissions/keeper/keeper.go

```diff
@@ -134,6 +146,9 @@ type Keeper struct {
 	// map of (topic, worker) -> forecast[]
 	forecasts collections.Map[collections.Pair[TopicId, ActorId], types.Forecast]
 
+	// map of (topic, reputer) -> reputer loss
+	lossBundles collections.Map[collections.Pair[TopicId, Reputer], types.ReputerValueBundle]
```

Comment:
Would consider calling this `topicLastReputerPayload`

Reply:
I followed the pattern that we were using to name the KVStore that we are using to save individual submissions: `inferences`, `forecasts` then `lossBundles`.